gems/pending/appliance_console/logging.rb

```diff
@@ -57,17 +57,15 @@ def self.log_filename
 
     # TODO: move say_error and say_info to prompting module?
     def say_error(method, output)
-      if interactive?
-        log = "\nSee #{ApplianceConsole::Logging.log_filename} for details." if ApplianceConsole::Logging.log_filename
-        text = "#{method.to_s.humanize} failed with error - #{output.truncate(200)}.#{log}"
-        say(text)
-        press_any_key
-        raise MiqSignalError
-      end
+      log = "\nSee #{ApplianceConsole::Logging.log_filename} for details." if ApplianceConsole::Logging.log_filename
+      text = "#{method.to_s.humanize} failed with error - #{output.truncate(200)}.#{log}"
+      say(text)
```

Comment:
This BZ was filed by  Luke Couzens, who reports to @dajohnso  I had discussed this with Luke Couzens. It is my understanding that they want output from appliance_console_cli so their automation does not blindly proceed when there are issues. From my conversations with Luke, I believe this is exactly why the BZ was filed.